setup.py

```diff
@@ -33,7 +33,7 @@ def get_version():
     ],
     'flask': ['Flask>=0.9'],
     'flask-login': ['Flask-Login>=0.2.9'],
-    'flask-sqlalchemy': ['Flask-SQLAlchemy>=1.0'],
+    'flask-sqlalchemy': ['Flask-SQLAlchemy>=1.0,<3.0.0'],
```

Comment:
Discarding V3.0.0 for Flask-Sqlalchemy as it's causing import [issue](https://github.com/kvesteri/sqlalchemy-continuum/actions/runs/3187069268/jobs/5198277223)

Reply:
Commit message doesn't give any information about above change. Shall we keep this in separate commit?

Reply:
Sure, raised PR in #4 for same and amended changes here